docs/user-guides/finetuner.md

```diff
@@ -0,0 +1,144 @@
+                ),
+            ],
+        ),
+    ]
+)
+train_da.push('name-of-train-data')
+```
+
+## Start Finetuner
+
+You may now create and run a fine-tuning job after login to Jina ecosystem.
+For simplicity, we directly use `clip-fashion-train-data` and `clip-fashion-eval-data` provided by Finetuner.
+
+```python
+import finetuner
+
+finetuner.login()
+run = finetuner.fit(
+    model='openai/clip-vit-base-patch32',
+    run_name='clip-fashion',
+    train_data='clip-fashion-train-data',
+    eval_data='clip-fashion-eval-data',  # optional
+    epochs=5,
+    learning_rate=1e-5,
+    loss='CLIPLoss',
+    cpu=False,
+)
+```
+
+After the job started, you may use [`.status`](https://finetuner.jina.ai/api/finetuner.run/#finetuner.run.Run.status) to check the status of the job.
+
+```python
+import finetuner
+
+finetuner.login()
+run = finetuner.get_run('clip-fashion')
+print(run.status())
+```
+
+When the status is `FINISHED`, you can download the tuned model to your local machine.
+
+```python
+import finetuner
+
+finetuner.login()
+run = finetuner.get_run('clip-fashion')
+run.save_artifact('clip-model')
+```
+
+You should now get a zip file containing the tuned model named `clip-fashion.zip` under the folder `clip-model`.
+
+## Use the Model
+
+After unzipping the model you get from the previous step, a folder with the following structure will be generated:
+
+```text
+.
+└── clip-fashion/
+    ├── config.yml
+    ├── metadata.yml
+    ├── metrics.yml
+    └── models/
+        ├── clip-text/
+        │   ├── metadata.yml
+        │   └── model.onnx
+        ├── clip-vision/
+        │   ├── metadata.yml
+        │   └── model.onnx
+        └── input-map.yml
+```
+
+Since the tuned model generated from Finetuner contains richer information such as metadata and config, we now transform it to simpler structure used by CLIP-as-service.
+
+First create a new folder named `clip-fashion-cas` or anything you like. This will be the storage of the models to use in CLIP-as-service.
+
+Second copy and move `clip-fashion/models/clip-text/model.onnx` to `clip-fashion-cas` and rename it to `textual.onnx`.
+
+Similarly, copy and move `clip-fashion/models/clip-vision/model.onnx` to `clip-fashion-cas` and rename it to `visual.onnx`.
+
+This is the expected structure of `clip-fashion-cas`:
+
+```text
+.
+└── clip-fashion-cas/
+    ├── textual.onnx
+    └── visual.onnx
+```
+
+In order to use finetuned model, create a custom YAML file `finetuned_clip.yml` like below. Learn more about [Flow YAML configuration](https://docs.jina.ai/fundamentals/flow/yaml-spec/) and [`clip_server` YAML configuration](https://clip-as-service.jina.ai/user-guides/server/#yaml-config).
+
+```yaml
+jtype: Flow
+version: '1'
+with:
+  port: 51000
+executors:
+  - name: clip_o
+    uses:
+      jtype: CLIPEncoder
+      metas:
+        py_modules:
+          - executors/clip_onnx.py
+      with:
+        name: ViT-B/32 # since finetuner only support ViT-B/32 for CLIP
```

Comment:
polish the comment (I suggest adding a warning phrase to emphasize this part). Remind the user that the model name should match the fine-tuned model. Otherwise, the output is not correct.

Reply:
warning make sense